.github/actions/install-linux-build-deps/action.yml

```diff
@@ -9,22 +9,16 @@ runs:
         sudo apt-get update
```

Comment:
Wondering changes in this file were needed in relation to workspaces or if you just optimized the deps we need to install here.
For the record this action was originally adapted from an install script and I wasn't aware at the time it could be optimized.

Reply:
There were some changes required to the linux install because we were using BuildJet's Ubuntu 22.04 runner which had some different deps requirements than whatever the version of Ubuntu used previously was.